app/renderer/js/pages/preference/shortcuts-section.ts

```diff
@@ -16,12 +16,8 @@ export default class ShortcutsSection extends BaseSection {
 		this.props = props;
 	}
 
-	// TODO - Deduplicate templateMacHTML and templateWinLinHTML functions. In theory
-	// they both should be the same the only thing different should be the userOSKey
-	// variable but there seems to be inconsistences between both function, one has more
-	// lines though one may just be using more new lines and other thing is the use of +.
-	templateMacHTML(): string {
-		const userOSKey = '⌘';
+	templateHTML(): string {
+		const OSKey = (process.platform === 'darwin') ? '⌘' : 'Ctrl';
```

Comment:
Upper-camel-case is typically reserved for class names in JavaScript. I’d call this `cmdOrCtrl`, which is consistent with the [Electron modifier name](https://www.electronjs.org/docs/api/accelerator#available-modifiers).